client/SDL/sdl_webview.cpp

```diff
@@ -47,7 +50,7 @@ class SchemeHandler : public QWebEngineUrlSchemeHandler
 				continue;
 
 			QByteArray code = pair[1].toUtf8();
-			*codeptr = (char*)calloc(1, code.size() + 1);
+			*codeptr = reinterpret_cast<char*>(calloc(1, code.size() + 1));
```

Comment:
Should be `static_cast` and return check is missing. This `char** codeptr` should be replaced with a `std::string` anyways.